code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -168,6 +168,15 @@
 	..()
 	reagents.add_reagent(ADMINORDRAZINE, 50)
 
+/obj/item/weapon/reagent_containers/pill/procizine
+	name = "Procizine pill"
+	desc = "What could possibly go wrong?"
+	icon_state = "pill6" //cyan-brown oblong
```

Comment:
This should definitely be green and purple.

Reply:
thanks doc